src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -450,6 +450,23 @@ private function deleteProject(Database $dbForConsole, callable $getProjectDB, c
             Query::equal('projectInternalId', [$projectInternalId])
         ], $dbForConsole);
 
+        // Delete VCS Installations, Repositories and Comments
+        $this->deleteByGroup('installations', [
+            Query::equal('projectInternalId', [$projectInternalId])
+        ], $dbForConsole, function (Document $document) use ($dbForConsole) {
```

Comment:
this will delete all the installations of that project ID also... not sure that is expected.